docs/admin_integration.rst

```diff
@@ -320,7 +320,7 @@ Add the following to ``CustomBookAdmin`` class (in ``tests/core/admin.py``)::
             kwargs.update({"author": form.cleaned_data.get("author", None)})
         return kwargs
 
-Then add the following to ``CustomBookAdmin`` class (in ``tests/core/admin.py``)::
+Then add the following to ``BookResource`` class (in ``tests/core/admin.py``)::
```

Comment:
I think it should be `EBookResource` if following the example in the documentation.

Reply:
Good point!
Fixed it also a few lines above.